### PR TITLE
Add new variable for machine names

### PR DESCRIPTION
**Pull request by hnrkndrssn**
Waiting for https://github.com/OctopusDeploy/OctopusDeploy/pull/1670 to be merged first

- [ ] Update **actual** version variable was introduced in

### CODE REVIEW

File: docs/deployment-process/variables/system-variables.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Deployment-level variables are drawn from the project and release being deployed
 |**`Octopus.Deployment.Id`** <br/>The ID of the deployment | *deployments-123*|
 |**`Octopus.Deployment.Name`** <br/>The name of the deployment | *Deploy to Production*|
 |**`Octopus.Deployment.PreviousSuccessful.Id`** <br/>The ID of the previous successful deployment of this project in the target environment | *deployments-122*|
-|**`Octopus.Deployment.Machines`** <br/>Ids of machines being targeted by the deployment. | machines-123,machines-124|
+!partial <deploymentMachineVariables>
```
Comment (droyad) [inline docs/deployment-process/variables/system-variables.md]: I think the current approach is to not use partials anymore, rather just note the version number. Check with @leecarlon

Comment (alfhenrik) [reply docs/deployment-process/variables/system-variables.md]: Oh, OK. I was not aware of that

Comment (leecarlon) [reply docs/deployment-process/variables/system-variables.md]: Rob's correct, but a bit ahead of the curve. We won't need to use partials for old versions when the new versioned version of the docs go live (ETA next week, with details to follow), and in fact, at that point, using partials will only really be useful for pre-release content.

Comment (alfhenrik) [reply docs/deployment-process/variables/system-variables.md]: OK, I'll revert it to just mention the version it was introduced in in the descripition

**Pull request closed without merging**